pages/services.vue

```diff
@@ -1,11 +1,35 @@
 <template>
-  <iframe
-    class="w-full h-full min-h-screen"
-    src="https://airtable.com/embed/shra12KjCQJwrnu8l?backgroundColor=gray&layout=card&viewControls=on"
-    frameborder="0"
-    onmousewheel=""
-    width="100%"
-    height="100%"
-    style="background: transparent; border: 0px solid #ccc"
-  ></iframe>
+  <div class="grid grid-cols-auto-fit gap-6 p-1">
+    <ul v-for="service in services" :key="service.id" role="list">
+      <TServiceListItem
+        :field-id="service.id"
+        :name="service.fields.Name"
+        :description="service.fields.Description"
+        :url="service.fields.URL"
+        :location="service.fields.Location"
+        :type="service.fields.Type"
+        :services="service.fields.Services"
+        :phone="service.fields.Phone"
+        :rating="service.fields.Rating"
+      />
+    </ul>
+  </div>
 </template>
+<script>
+import TServiceListItem from '../components/services/TServiceListItem.vue'
```

Comment:
Nuxt automatically imports any components in your components/ directory (along with components that are registered by any modules you may be using).

https://v3.nuxtjs.org/guide/directory-structure/components